#! /bin/sh /usr/share/dpatch/dpatch-run
## rm_bogus_shebang.dpatch by Stefano Zacchiroli <zack@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Remove python shebang.

@DPATCH@
diff -urNad trunk~/SOAPpy/wstools/Namespaces.py trunk/SOAPpy/wstools/Namespaces.py
--- trunk~/SOAPpy/wstools/Namespaces.py	2005-02-09 16:21:03.000000000 +0100
+++ trunk/SOAPpy/wstools/Namespaces.py	2007-03-21 10:14:43.000000000 +0100
@@ -1,4 +1,3 @@
-#! /usr/bin/env python
 """Namespace module, so you don't need PyXML 
 """
 
diff -urNad trunk~/SOAPpy/wstools/__init__.py trunk/SOAPpy/wstools/__init__.py
--- trunk~/SOAPpy/wstools/__init__.py	2005-02-09 16:21:02.000000000 +0100
+++ trunk/SOAPpy/wstools/__init__.py	2007-03-21 10:14:38.000000000 +0100
@@ -1,4 +1,3 @@
-#! /usr/bin/env python
 """WSDL parsing services package for Web Services for Python."""
 
 ident = "$Id: __init__.py,v 1.11 2004/12/07 15:54:53 blunck2 Exp $"
diff -urNad trunk~/SOAPpy/wstools/c14n.py trunk/SOAPpy/wstools/c14n.py
--- trunk~/SOAPpy/wstools/c14n.py	2005-02-09 16:21:03.000000000 +0100
+++ trunk/SOAPpy/wstools/c14n.py	2007-03-21 10:14:35.000000000 +0100
@@ -1,4 +1,3 @@
-#! /usr/bin/env python
 """Compatibility module, imported by ZSI if you don't have PyXML 0.7.
 
 No copyright violations -- we're only using parts of PyXML that we
diff -urNad trunk~/SOAPpy/wstools/logging.py trunk/SOAPpy/wstools/logging.py
--- trunk~/SOAPpy/wstools/logging.py	2005-02-09 16:21:03.000000000 +0100
+++ trunk/SOAPpy/wstools/logging.py	2007-03-21 10:14:41.000000000 +0100
@@ -1,4 +1,3 @@
-#! /usr/bin/env python
 """Logging"""
 import sys
 
